server/src/controllers/Events/resolver.ts

```diff
@@ -511,7 +511,13 @@ export class EventResolver {
     await updateWaitlistForUserRemoval({ event, userId: ctx.user.id });
 
     const updatedEventUser = await prisma.event_users.update({
-      data: { rsvp: { connect: { name: 'no' } } },
+      data: {
+        rsvp: { connect: { name: 'no' } },
+        ...(eventUser.subscribed && {
+          subscribed: false,
+        }),
```

Comment:
```suggestion
        subscribed: false,
```
This is basically the same since if `subscribed` is `false`, then setting to `false` again does nothing.